From b7b6ac6462dcef7274f1050e08bf841ec096d15d Mon Sep 17 00:00:00 2001
From: Sanjeev Gupta <ghane0@gmail.com>
Date: Thu, 1 Dec 2016 02:17:36 +0800
Subject: [PATCH 088/268] Remove repeated lines in discover.html ,

and fix an anchor.

The text for the Multicast needs to be copy-edited, and
revised by someone who can confirm this works.
---
 docs/discover.txt         | 21 +++++++++------------
 docs/includes/confopt.txt |  2 +-
 2 files changed, 10 insertions(+), 13 deletions(-)

diff --git a/docs/discover.txt b/docs/discover.txt
index e419568..b06948b 100644
--- a/docs/discover.txt
+++ b/docs/discover.txt
@@ -169,23 +169,23 @@ command but with a multicast (IPv4 class _D_ or IPv6 prefix _FF_)
 group address. The IANA has designated IPv4 address 224.1.1.1 and IPv6
 address FF05::101 (site local) for NTP.
 
+A manycast server is configured using the +manycastserver+ command,
+which listens on the specified address for manycast client
+messages.  If a manycast server is in scope of the current TTL and is
+itself synchronized to a valid source and operating at a stratum level
+equal to or lower than the manycast client, it replies with an ordinary
+unicast server message.
+
 The client sends ordinary client mode messages, but to one of
-these broadcast addresses rather than a unicast address, and sends
+these muticast addresses rather than a unicast address, and sends
 only if less than +maxclock+ associations remain and then only at the
 minimum feasible rate and minimum feasible time-to-live (TTL)
 hops. The polling strategy is designed to reduce as much as possible
-the volume of broadcast messages and the effects of implosion due to
+the volume of manycast client messages and the effects of implosion due to
 near-simultaneous arrival of manycast server messages. There can be as
 many manycast client associations as different addresses, each one
 serving as a template for future unicast client/server associations.
 
-A manycast server is configured using the +manycastserver+ command,
-which listens on the specified broadcast address for manycast client
-messages.  If a manycast server is in scope of the current TTL and is
-itself synchronized to a valid source and operating at a stratum level
-equal to or lower than the manycast client, it replies with an ordinary
-unicast server message.
-
 The manycast client receiving this message mobilizes a preemptable
 client association according to the matching manycast client template.
 This requires the server to be cryptographically authenticated and the
@@ -201,9 +201,6 @@ the "best" associations according to stratum and synchronization
 distance. The surviving associations then continue in ordinary
 client/server mode.
 
-The manycast client polling strategy is designed to reduce as much as
-possible the volume of manycast client messages and the effects of
-implosion due to near-simultaneous arrival of manycast server messages.
 The strategy is determined by the _manycastclient_, _tos_ and _ttl_
 configuration commands. The manycast poll interval is normally eight
 times the system poll interval, which starts out at the _minpoll_ value
diff --git a/docs/includes/confopt.txt b/docs/includes/confopt.txt
index 528f719..f46ef8c 100644
--- a/docs/includes/confopt.txt
+++ b/docs/includes/confopt.txt
@@ -5,6 +5,6 @@
 * link:confopt.html#manycastclient[manycastclient - configure manycast client  association]
 * link:confopt.html#pool[pool - configure pool association]
 * link:confopt.html#unpeer[unpeer - remove association]
-* link:confopt.html#manycastserver[manycastserver - enable manycast server]
+* link:discover.html#mcst[manycastserver - enable manycast server]
 * link:comdex.html[Command Index]
 
-- 
2.7.4

